tests/foreman/cli/test_contentview.py

```diff
@@ -4794,6 +4794,29 @@ def test_positive_incremental_update_propagate_composite(self):
         )
 
 
+    @tier3
```

Comment:
There are two empty lines above, while there should be only one. This causes flake8 to report error, and travis to fail the build (at least I think that causes travis to fail).

Two empty lines below this test are ok, as these are between method and next class.